stores.js

```diff
@@ -7,6 +7,8 @@ export const gjScheme = writable(emptyGeojson());
 export const currentSidebarHover = writable(null);
 export const currentMapHover = writable(null);
 
+export const openFromSidebar = writable(null);
```

Comment:
I thought about using event dispatchers, but the flow would be too complicated. InterventionList up to App, but then down to DrawControls and EditingLayer. Doesn't seem to be a nice way of doing that, so we can just use stores.

Reply:
I would be lying if I said I knew what that means. Trust your judgement here!